File: Lab 1.1/C++/String.cpp
#include <algorithm>
#include "String.h"

std::string StringManipulator::toUppercase(std::string input) 
{
    for (char &c : input) c = std::toupper(c);
    return input;
}

std::string StringManipulator::replaceLetters(std::string input, char oldLetter, char newLetter) 
{
    std::replace(input.begin(), input.end(), oldLetter, newLetter);
    return input;
}

size_t StringManipulator::getLength(std::string input) 
{
    return input.length();
}

File: Lab 1.1/C#/String.cs
using System;
using Console_application;

namespace String_manipulator {

public class MainClass {
    public void Main(string input, string oldletter, string newletter)  
    {
        string input_Uppercase = input.ToUpper();
        int input_Length = input.Length;
        string input_Replace = input.Replace(oldletter, newletter);

        Program transfer = new Program();
        transfer.Output(input, input_Length, input_Uppercase, input_Replace);
    }
  }
}   